src/components/tutorial/ArraysTutorial.tsx

```diff
@@ -23,6 +36,47 @@ const ArraysTutorial: React.FC<ArraysTutorialProps> = (props) => {
     console.log(example.propertyOne);
     console.log(example.methodOne());
 
+export const ArraysTutorial: React.FC = () => {
+  const [list, setList] = useState<TodoItem[]>([{id: 0, value: ''}])
+
+  const handleChange = (value: string, id: TodoItem['id']) => {
```

Comment:
It is a bit strange to only pass the ID to this function instead of the entire item, you never know if you need the entire item or how complex would the function become, but this looks good